### PR TITLE
feat(launchpad): add pluralize lib

### PR DESCRIPTION
**Pull request by WaDadidou**
Add the lib `pluralize` and `@types/pluralize` to handle plural strings.
And remove `pluralOrNot` function

### CODE REVIEW

File: packages/utils/text.tsx [hunk 1]
```diff
@@ -34,4 +34,10 @@ export const replaceBetweenString = (
   `${origin.substring(0, startIndex)}${insertion}${origin.substring(endIndex)}`;
 
 export const pluralOrNot = (word: string, quantity: number) =>
```
Comment (hthieu1110) [inline packages/utils/text.tsx:36]: I think for most of cases , this is fine if we don't need the perfect solution :D 
But if we are perfectionist, we have to handle many edge cases like words finished by s (like gas) or ch ...

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Review by MikaelVallenet — DISMISSED** [review]

File: packages/utils/text.tsx (hunk 1 above)
Comment (WaDadidou) [reply packages/utils/text.tsx:36]: Yes, I added the lib `pluralize` => https://classic.yarnpkg.com/en/package/pluralize

**Review by Villaquiranm — APPROVED** [review]
LGTM :)

**Review by n0izn0iz — APPROVED** [review]

**Review by MikaelVallenet — APPROVED** [review]

**Pull request merged**